rpms/SPECS/4.2.6/wazuh-manager-4.2.6.spec

```diff
@@ -830,6 +830,8 @@ rm -fr %{buildroot}
 
 
 %changelog
+* Mon Mar 25 2022 support <info@wazuh.com> - 4.2.6
```

Comment:
Correct the date.

Reply:
Done 6318a8a96caf59fd477451b2802b4f8c620b3eb6